source/configuration_manual/config_file/config_variables.rst

```diff
@@ -100,6 +100,14 @@ Mail service user variables:
 |          |                |                                                               |
 |          |                | .. versionadded:: v2.2.19                                     |
 +----------+----------------+---------------------------------------------------------------+
+|          | local_ip       | local IP address (alias of %l)                                |
+|          |                |                                                               |
```

Comment:
https://github.com/dovecot/documentation/pull/165 wasn't merged, but v2.3.12 should be replaced with v2.3.13 in those places..

Reply:
included a commit with correct version fix.